.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -19,13 +19,16 @@ on:
   workflow_dispatch:
     inputs:
       build-branch:
-        description: 'Build a custom image of Rawls from the specified branch. Default is true; if false, the tests run against the version specified by bee-version-template.'
+        description: 'Build a custom image of Rawls from the specified branch. Default is false, meaning tests are run against the version specified by bee-version-template.'
         required: true
-        default: true
-        type: boolean
```

Comment:
Declaring a boolean type leads to misery because github actions don't support them well. They get turned into strings, making comparisons error-prone.

Going with a choice makes it clear what the values will be.

Reply:
Looks like this:

<img width="366" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/1a91d65a-9ef8-40c6-9f3c-ad3f5ca03412">